var/spack/repos/builtin/packages/fpart/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Comment:
We haven't switched to enforcing 2023 yet.  Can you change this to 2022 so I can merge it?

Reply:
Yes, switched.